test/test_importer.py

```diff
@@ -414,7 +414,7 @@ class ImportTarTest(ImportZipTest):
     def create_archive(self):
         (handle, path) = mkstemp(dir=self.temp_dir)
         os.close(handle)
-        archive = TarFile(path, mode='w')
+        archive = TarFile(py3_path(path), mode='w')
```

Comment:
These uses of `py3_path` look right to me. Are there any lingering questions here?